src/types/timestamp.rs

```diff
@@ -48,7 +46,7 @@ pub enum Mantissa {
     Digits(u32),
     /// Specifies the fractional seconds precisely as a `Decimal` in the range `>= 0` and `< 1`.
     /// The Decimal will have the correct precision; the complete value can and should be used.
-    /// This representation should only be used for precisions greater than nanoseconds as can
+    /// This representation should only be used for precisions greater than nanoseconds as it can
```

Comment:
> it can require allocations

As is "the implementation may be changed without warning such that it allocates"?

Reply:
RE: "without warning", this is not a public-facing type. (Its host module is private.)